### PR TITLE
Chart, support for disabling RBAC

### PR DESCRIPTION
**Pull request by migmartri**
If you try to install RBAC resources in a cluster that has RBAC disabled from a service account (i.e Tiller serviceaccount) the installation fails with the following error

```
Error: release kubeapps failed: clusterroles.rbac.authorization.k8s.io "kubeapps-apprepository-controller" is forbidden: attempt to grant extra privileges: [{[get] [batch] [cronjobs] [] []} {[list] [batch] [cronjobs] [] []} {[watch] [batch] [cronjobs] [] []}] user=&{system:serviceaccount:kube-system:default 63e12d0a-9f47-11e8-bb6c-d6e551b314a5 [system:serviceaccounts system:serviceaccounts:kube-system system:authenticated] map[]} ownerrules=[] ruleResolutionErrors=[]
```

This can be prevented by skipping these resources during installation. It seems to be a common pattern in [the upstream charts](https://github.com/helm/charts/search?q=rbac.create&unscoped_q=rbac.create) to have a rbac.create flag in the values.yaml set to true by default. 

Fixes https://github.com/kubeapps/kubeapps/issues/480

### CODE REVIEW

File: chart/kubeapps/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubeapps
-version: 0.1.0
+version: 0.1.2
```
Comment (migmartri) [inline chart/kubeapps/Chart.yaml]: This PR depends on https://github.com/kubeapps/kubeapps/pull/484 that's why it bumps twice.

File: chart/kubeapps/README.md [hunk 2]
```diff
@@ -170,3 +170,19 @@ For annotations, please see [this document](https://github.com/kubernetes/ingres
...
+
+### Forbidden error while installing the Chart
+
+If during installation you run into an error similar to:
+
+```
+Error: release kubeapps failed: clusterroles.rbac.authorization.k8s.io "kubeapps-apprepository-controller" is forbidden: attempt to grant extra privileges: [{[get] [batch] [cronjobs] [] []...
+```
+
+It is possible that your cluster does not have Role Based Access Control (RBAC) fully configured. In which case you should perform the chart installation by setting `rbac.create=false`
```
Comment (prydonius) [inline chart/kubeapps/README.md]: let's add a colon at the end of this sentence

File: chart/kubeapps/values.yaml [hunk 3]
```diff
@@ -170,3 +170,8 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+## For RBAC support:
```
Comment (prydonius) [inline chart/kubeapps/values.yaml]: Use a single has to be consistent with the rest of the comments in this file

File: chart/kubeapps/values.yaml [hunk 4]
```diff
@@ -170,3 +170,8 @@ nodeSelector: {}
...
+rbac:
+  # Perform creation of rbac resources
```
Comment (prydonius) [inline chart/kubeapps/values.yaml]: uppercase RBAC

**Pull request merged**